lib/thrift/protocol/binary.ex

```diff
@@ -86,7 +90,13 @@ defmodule Thrift.Protocol.Binary do
     byte_size(name)::32-signed, name::binary, sequence_id::32-signed>>
   end
 
-  def deserialize(:message_begin,<<1::size(1), 1::size(15), _::size(8),
+  @doc """
+  Deserializes a Thrift-encoded binary.
+  """
+  @spec deserialize(atom, binary) ::
+    {:ok, {atom, integer, binary, binary}} |
```

Comment:
Might be helpful to give these names?  At first glance it's not obvious to me what the values are.